src/test/java/seedu/address/logic/parser/FindCommandParserTest.java

```diff
@@ -1,34 +1,33 @@
-// package seedu.address.logic.parser;
-//
-// import static seedu.address.commons.core.Messages.MESSAGE_INVALID_COMMAND_FORMAT;
-// import static seedu.address.logic.parser.CommandParserTestUtil.assertParseFailure;
-// import static seedu.address.logic.parser.CommandParserTestUtil.assertParseSuccess;
-//
-// import java.util.Arrays;
-//
-// import org.junit.jupiter.api.Test;
-//
-// import seedu.address.model.person.NameContainsKeywordsPredicate;
-//
-// public class FindCommandParserTest {
-//
-//     private FindCommandParser parser = new FindCommandParser();
-//
-//     @Test
-//     public void parse_emptyArg_throwsParseException() {
-//         assertParseFailure(parser, "     ", String.format(MESSAGE_INVALID_COMMAND_FORMAT,
-//         FindCommand.MESSAGE_USAGE));
-//     }
-//
-//     @Test
-//     public void parse_validArgs_returnsFindCommand() {
-//         // no leading and trailing whitespaces
-//         FindCommand expectedFindCommand =
-//                 new FindCommand(new NameContainsKeywordsPredicate(Arrays.asList("Alice", "Bob")));
-//         assertParseSuccess(parser, "Alice Bob", expectedFindCommand);
-//
-//         // multiple whitespaces between keywords
-//         assertParseSuccess(parser, " \n Alice \n \t Bob  \t", expectedFindCommand);
-//     }
-//
-// }
+package seedu.address.logic.parser;
+
+import static seedu.address.commons.core.Messages.MESSAGE_INVALID_COMMAND_FORMAT;
+import static seedu.address.logic.parser.CommandParserTestUtil.assertParseFailure;
+import static seedu.address.logic.parser.CommandParserTestUtil.assertParseSuccess;
+
+import java.util.List;
+
+import org.junit.jupiter.api.Test;
+
+import seedu.address.logic.commands.FindCommand;
+import seedu.address.logic.parser.exceptions.ParseException;
+import seedu.address.model.recipe.RecipeContainsKeywordPredicate;
+
+class FindCommandParserTest {
+
+    private final FindCommandParser parser = new FindCommandParser();
+
+    @Test
+    public void parse_emptyArg_throwsException() {
+        assertParseFailure(parser, " ",
+                String.format(MESSAGE_INVALID_COMMAND_FORMAT, FindCommand.MESSAGE_USAGE));
+    }
+
+    @Test
+    public void parse_validArgs_returnsFindCommand() throws ParseException {
+        RecipeContainsKeywordPredicate pred = new RecipeContainsKeywordPredicate(List.of("Western", "Bacon"));
+        FindCommand expected = new FindCommand(pred);
+
+        assertParseSuccess(parser, "Western, Bacon", expected);
+        assertParseSuccess(parser, "      Western \r\n\t\t,    Bacon", expected);
```

Comment:
Minor: should an input that includes newline breaks be considered valid? am not too sure bout this (it's a v small edge case tho)

Reply:
For this part, the Parser would comma-separate the values to "      Western \r\n\t\t", and "    Bacon". The point of that assertion is to ensure that the white spaces of each CSV is being trimmer to prevent any unexpected behaviours.